src/IROperator.h

```diff
@@ -820,7 +885,12 @@ inline Expr pow(Expr x, Expr y) {
     if (x.type() == Float(64)) {
         y = cast<double>(y);
         return Internal::Call::make(Float(64), "pow_f64", {x, y}, Internal::Call::Extern);
-    } else {
+    }
+    else if (x.type() == Float(16)) {
+         y = cast<float16_t>(y);
+        return Internal::Call::make(Float(16), "pow_f16", {x, y}, Internal::Call::Extern);
```

Comment:
I was thinking of these: https://software.intel.com/en-us/node/524287

There's probably an llvm intrinsic to get it to emit the corresponding instructions. Pre-avx2 we'd have to think of some clever way to emulate it with vectorizable bit tricks.